src/analysis/normed/group/basic.lean

```diff
@@ -738,6 +738,12 @@ lipschitz_with_one_norm'.uniform_continuous
 lemma uniform_continuous_nnnorm' : uniform_continuous (λ (a : E), ∥a∥₊) :=
 uniform_continuous_norm'.subtype_mk _
 
+@[to_additive] lemma norm_eq_zero_iff_closure_one {x : E} : ∥x∥ = 0 ↔ x ∈ closure ({1} : set E) :=
```

Comment:
IMHO, RHS is simpler than LHS.

Reply:
I'm okay with swapping the arguments. I use `metric.closed_ball_zero'` in the proof, but I think this is a useful specialization.